plugins/cros-ec/fu-cros-ec-firmware.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2020 Benson Leung <bleung@chromium.org>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include "fu-firmware.h"
+#include "fu-fmap-firmware.h"
+
+#define FU_TYPE_CROS_EC_FIRMWARE (fu_cros_ec_firmware_get_type ())
+G_DECLARE_FINAL_TYPE (FuCrosEcFirmware, fu_cros_ec_firmware, FU, CROS_EC_FIRMWARE, FuFirmware)
```

Comment:
I sorta expected that this would actually derive from fmap firmware, shouldn't it?

Reply:
Good catch. Done.